Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -73,27 +72,27 @@ class TransactionHistoryViewController: UIViewController {
     }
     
     @objc func handleRefresh(_ refreshControl: UIRefreshControl) {
-        self.loadTransactions()
+        self.reloadTxsForCurrentFilter()
     }
     
-    func loadTransactions() {
-        self.mainContens.removeAll()
-        self.Filtercontent.removeAll()
+    func loadTransactions(filter: Int? = 0) {
+        self.transactions.removeAll()
+        var error: NSError?
+        let allTransactions = AppDelegate.walletLoader.wallet?.getTransactions(0, txFilter: Int32(filter!), error: &error)
+        if error != nil{
+            print(error!.localizedDescription)
```

Comment:
Add context to the error message. Simply printing the error directly may not be helpful for debugging.